verifiers/internal/gha/rekor.go

```diff
@@ -123,14 +124,19 @@ func extractCert(e *models.LogEntryAnon) (*x509.Certificate, error) {
 
 	var publicKeyB64 []byte
 	switch e := eimpl.(type) {
-	case *intotod.V001Entry:
+	case *intoto_v001.V001Entry:
```

Comment:
ahh, I commented too quickly :) Backward compatibility is done here. So I think this PR should be ready to merge.

Reply:
Yep, this should be fully backwards compatible, this just adds the additional DSSE Rekor type. We could and should probably make another change to add the intoto v002 type for completeness.